kolibri/core/assets/src/views/coach-content-label.vue

```diff
@@ -33,9 +33,12 @@
       },
     },
     computed: {
+      valueText() {
+        return this.$formatNumber(this.value);
```

Comment:
I _think_ the more correct fix might actually be to use the [number type](https://formatjs.io/guides/message-syntax/#number-type) in ICU syntax like:

```
topicTitle: 'Contains {count, number} {count, plural, one {coach resource} other {coach resources}}',
```

@rtibbles I think you added the `$formatNumber` function? Do you know under what circumstances it's appropriate to use? (And then we can document it in our [i18n docs](http://kolibri-dev.readthedocs.io/en/develop/references/i18n.html).)

Reply:
`$formatNumber` is the vue-intl method for just formatting a number (in the case where it is used outside of any particular text string). As you note, if used within the context of a string, then the ICU syntax should be used (which ultimately will do, in essence, the `$formatNumber` call under the hood during the string interpolation).

Reply:
Hmm... looks like I might be wrong. 

Searching the code for the string `, number}` returns bunch of examples that don't get properly internationalized. For instance the setup wizard:

```javascript
    $trs: {
      progressIndicator: 'Step {currentStep, number} of {totalSteps, number}',
    },
```

![image](https://user-images.githubusercontent.com/2367265/42540919-375fdc64-8455-11e8-817b-99094f63b338.png)

This to seems fairly problematic to me ...

Reply:
https://github.com/learningequality/vue-intl/issues/17

Reply:
It also seems like we need to vendor ui-textbox in order to internationalize the `0/50` also.

Reply:
> It also seems like we need to vendor ui-textbox in order to internationalize the `0/50` also.

yes, I think that's right (can also submit a patch upstream). Had #3141 open for that already

> I think the `valueText` computed string is still necessary, since the coach label is basically just `<icon> <number>`, and the number needs to be localized.

There are a fair number of strings in the code that are only a localized number, with no text to translate - for example:

https://github.com/learningequality/kolibri/blob/0aa7dabaa28059416691f5d9f5cb63696a33910c/kolibri/plugins/coach/assets/src/views/exams/exam-report-page/index.vue#L229

However these strings might be confusing for translators, so I can see an argument for using `$formatNumber` instead. In either case we should try to be consistent.

Does `$formatNumber` support all the modifiers and variations that ICU syntax supports?